network/server.go

```diff
@@ -377,14 +408,35 @@ func (s *Server) hasPeer(peerID peer.ID) bool {
 }
 
 func (s *Server) numOpenSlots() int64 {
```

Comment:
Please rename this method to something more clear, since it returns the number of free outbound connections

Reply:
Done, commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/d9d37fe56f78bdd5d13b4d90975882044801b0be)